### DISCUSSION TITLE
Make flow names & descriptions translatable

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#24910] [Feature Requests]
### Describe the Improvement

Since the translations functionality within the app mostly everything can be labeled with translatable content. One exceptions are flows:
Currently it's not possible to use an app-translation for the flows name and description.

To solve this, the following files would need to be updated:
- [Flow overview](https://github.com/directus/directus/blob/5f73ccd1f2b4ea3b1c2031507732f056abc72bd9/app/src/modules/settings/routes/flows/overview.vue)
- [Flow drawer](https://github.com/directus/directus/blob/5f73ccd1f2b4ea3b1c2031507732f056abc72bd9/app/src/modules/settings/routes/flows/flow-drawer.vue)
- [Flow sidebar component
](https://github.com/directus/directus/blob/5f73ccd1f2b4ea3b1c2031507732f056abc72bd9/app/src/views/private/components/flow-sidebar-detail.vue#L178)

**Comment by kevincam3** [comment]
I would like to try and tackle this issue. I have a question. When looking at the translations options I see two different ways this could be done and I'm not sure if one is a newer version or if there's a reason there are two different way. But which one of these ways should it be done.
![image](https://github.com/user-attachments/assets/3569ba75-6d0f-4575-9ad4-17d3d0606090)

![image](https://github.com/user-attachments/assets/a35cb026-8de5-4efb-9afd-55be4b05ce7f)
